collectors/ebpf.plugin/README.md

```diff
@@ -241,69 +322,100 @@ filesystem, the collector needs to attach `kprobes` and `kretprobes` for each on
 
 #### File descriptor
 
-To give metrics related to `open` and `close` events, instead of attaching kprobes for each syscall used to do these 
+To give metrics related to `open` and `close` events, instead of attaching kprobes for each syscall used to do these
 events, the collector attaches `kprobes` for the common function used for syscalls:
 
-- [do_sys_open](https://0xax.gitbooks.io/linux-insides/content/SysCall/linux-syscall-5.html ): Internal function used to 
+- [`do_sys_open`](https://0xax.gitbooks.io/linux-insides/content/SysCall/linux-syscall-5.html ): Internal function used to
    open files.
-- [do_sys_openat2](https://elixir.bootlin.com/linux/v5.6/source/fs/open.c#L1162): Function called from do_sys_open since 
-   version `5.6.0`.
-- [close_fd](https://www.mail-archive.com/linux-kernel@vger.kernel.org/msg2271761.html): Function used to close file 
+- [`do_sys_openat2`](https://elixir.bootlin.com/linux/v5.6/source/fs/open.c#L1162):
+  Function called from `do_sys_open` since version `5.6.0`.
+- [`close_fd`](https://www.mail-archive.com/linux-kernel@vger.kernel.org/msg2271761.html): Function used to close file
   descriptor since kernel `5.11.0`.
 - `__close_fd`: Function used to close files before version `5.11.0`.
 
 #### File error
 
 This chart shows the number of times some software tried and failed to open or close a file descriptor.
 
-#### VFS Deleted objects 
+#### VFS
+
+The Linux Virtual File System (VFS) is an abstraction layer on top of a
+concrete filesystem like the ones listed in the parent section, e.g. `ext4`.
+
+In this section we list the mechanism by which we gather VFS data, and what
+charts are consequently created.
+
+##### VFS eBPF Hooks
+
+To measure the latency and total quantity of executing some VFS-level
+functions, ebpf.plugin needs to attach kprobes and kretprobes for each of the
+following functions:
+
+- `vfs_write`: Used for monitoring the number of successful & failed filesystem
```

Comment:
Let's keep the list syntax consistent through out. 

```suggestion
- `vfs_write`: Function used for monitoring the number of successful & failed filesystem
```

Reply:
https://github.com/netdata/netdata/pull/11752/commits/889eae75c14e167cf7c51e91ee947a172d505351